events.go

```diff
@@ -107,6 +107,14 @@ func (s *Session) handleEvent(framer *framer) {
 }
 
 func (s *Session) handleSchemaEvent(frames []frame) {
+	for _, frame := range frames {
```

Comment:
It looks like there are cases where the sessions `schemaDescriber` is nil, we should check and not clear metadata.